pkg/storage/bloom/v1/bloom_tokenizer.go

```diff
@@ -120,6 +120,16 @@ func (bt *BloomTokenizer) Populate(
 	if next {
 		// The last bloom has been made available via the `Next()` call above
 		bloom = blooms.At()
+
+		// TODO(salvacorts): Delete this once we solve the correctness bug
+		// We noticed some blooms are empty on the resulting blocks.
+		// We have the feeling that the empty blooms may be reused from old blocks.
+		// Here we log an error if we find an empty bloom.
+		if bloom.Count() == 0 {
+			level.Warn(bt.logger).Log(
+				"msg", "Found existing empty bloom",
```

Comment:
messages should start with lower case letters
```suggestion
				"msg", "found existing empty bloom",
```